run.sh

```diff
@@ -63,8 +72,10 @@ if git clone $FONTC_REPO $FONTC_DIR ; then
     ( $SCRIPT_PATH )
     cd ..
 fi
+
 echo "cleaning up"
 rm -rf $FONTC_DIR
+rm $LOCKFILE
```

Comment:
every case where you can exit above needs to rm lockfile. You could ease the pain by adding functions.

Reply:
oh yea total brainfart. moved to a function, calling it where we exit.